contracts/TokenConversionLib.sol

```diff
@@ -0,0 +1,59 @@
+pragma solidity 0.4.23;
```

Comment:
Please add a header consistent with the headers in our other Solidity files.

Reply:
Please fix for all files.